.travis.yml

```diff
@@ -32,20 +32,31 @@ install:
   - pip install -r rootfs/dev_requirements.txt
   - make -C docs/ deps
   - GLIDE_HOME=/home/travis/.glide make -C client bootstrap
+  - make prep-bintray-json
 script:
   - make test
   - make -C client/ build test
   - make -C docs/ build test
 deploy:
-  provider: script
-  # ensure client/doc builds aren't removed
-  # see https://docs.travis-ci.com/user/deployment/#Uploading-Files
-  skip_cleanup: true
-  script: _scripts/deploy.sh
-  on:
-    branch: master
+  - provider: script
+    # ensure client/doc builds aren't removed
+    # see https://docs.travis-ci.com/user/deployment/#Uploading-Files
+    skip_cleanup: true
+    script: _scripts/deploy.sh
+    on:
+      branch: master
+  - provider: bintray
+    # ensure client/doc builds aren't removed
+    # see https://docs.travis-ci.com/user/deployment/#Uploading-Files
```

Comment:
I kept them there as an intention that both deploy scripts do the same thing, but I can probably remove it as a duplicate.